ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -48,23 +79,62 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
         <style>
           :host {
             @apply --layout-vertical;
+            transition: all 0.7s;
           }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 6px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          nuxeo-collapsible {
+            --nuxeo-collapsible-heading-padding: 0;
+          }
+          label {
+            @apply --nuxeo-label;
+          }
```

Comment:
This raises an interesting question.
How do we want to approach this kind of situations from now on? Should we move some of our styling to elements? 🙂 

Currently, this is defined at WebUI level and being applied on storybook "demo theme".

Reply:
This isn't really "defined" anywhere, where basically just applying a `--nuxeo-label` mixin that may exist or not.
In the previous demos we had a demo theme, see https://github.com/nuxeo/nuxeo-elements/blob/master/ui/demo/nuxeo-demo-theme. Even if this means it may not be consistent with Web UI it showcases how we customize the elements. Ideally the elements would document they're usage of the mixins too.

Reply:
And with our current demos we have a `demo-theme` too.  (see [here](https://github.com/nuxeo/nuxeo-elements/blob/master/storybook/.storybook/nuxeo-demo-theme.js)) 🙂 

My question arises  after the discussion and decision made under button style improvements made recently (https://github.com/nuxeo/nuxeo-elements/pull/227).
In essence, the inner question is the same. Which kind of look & feel we want by default...

Reply:
For other elements, when not used internally by one of our elements, we don't really care. For Nuxeo elements we'd like them to look as close as possible to WebUI ootb which can be done with proper default values for CSS properties but I guess things are trickier with style mixins 🤔